cdc/log.cc

```diff
@@ -389,12 +390,24 @@ bytes log_data_column_deleted_elements_name_bytes(const bytes& column_name) {
     return to_bytes(cdc_deleted_elements_column_prefix) + column_name;
 }
 
+// Makes sure that number of sstables won't grow too large
+static int calculate_window_size(int ttl) {
+    // What's the minimum window that won't create more than 24 sstables.
+    auto window_size = std::chrono::seconds(ttl / 24 + (ttl % 24 == 0 ? 0 : 1));
```

Comment:
It rounds up if ttl does not divide by 24 with no remainder. For example if ttl is 50 then window size shouldn't be 2 but 3 because 50 / 2 > 24 but 50 / 3 <= 24.

Always adding +1 would be less efficient. For example if ttl is 48 then it would lead to window size of 3 while 2 would be perfectly good.